src/plugins/dev_tools/public/plugin.ts

```diff
@@ -125,10 +127,19 @@ export class DevToolsPlugin implements Plugin<DevToolsSetup> {
     };
   }
 
-  public start() {
+  public start(core: CoreStart) {
     if (this.getSortedDevTools().length === 0) {
       this.appStateUpdater.next(() => ({ navLinkStatus: AppNavLinkStatus.hidden }));
     }
+    core.chrome.navControls.registerRightNavigation({
+      // order of dev tool should be after advance settings
+      order: 2,
```

Comment:
Maybe we should put the orders into a common constant file so that other maintainers are easy to tell who sits before who.